app/src/main/java/org/simple/clinic/home/patients/PatientsEffect.kt

```diff
@@ -2,12 +2,16 @@ package org.simple.clinic.home.patients
 
 sealed class PatientsEffect
```

Comment:
`Patients` is a very generic name. We should think of renaming this later.

Reply:
Yes, I was wondering what to name this, but it's the `Patients` tab in the home screen so it ended up getting called the same.

I wanted to call it the `HomeScreen` but we already have a home screen to set up tabs so I decided to leave it as is for now. 😅

Reply:
Maybe `PatientsTab`?

Reply:
Better than what we have. Will make this change in the final PR.